locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -197,9 +197,11 @@ msgstr "Search"
 msgid "basicsearchform.placeholder"
 msgstr "Search..."
 
+
+
 #: ds_judgements_public_ui/templates/includes/search_term_component.html
-msgid "basicsearchform.helptext"
-msgstr "Enter keyword, party name or neutral&nbsp;citation"
+msgid "basic.search.helptext"
```

Comment:
```suggestion
msgid "search.basic.helptext"
```

This should probably go from least-specific to most-specific.

Reply:
aha, agree, but maybe this is a thing we should look at across the whole site? our translation keys are somewhat improvised everywhere at the moment, and it'd be good to give them a more consistent heirarchy! I'll add a card to look at this as a refactoring if that's ok!